node/flags/string_to_uint64.go

```diff
@@ -59,7 +59,7 @@ func (s *StringToUint64Value) String() string {
 // Nil is returned if value fails format validation (consistnt with viper behavior).
 func CastStringToMapStringUint64(value string) map[string]uint64 {
```

Comment:
This function isn't actually used anywhere besides in a single test and there I think it could just be inlined 🙂